lib/signable/concerns/query.rb

```diff
@@ -34,7 +34,7 @@ def create
       end
 
       module ClassMethods
-        def all(offset = 0, limit = 10)
+        def all(offset: 0, limit: 30)
```

Comment:
Usually when positional arguments are switched to key word, the invocation of the method needs to be changed as well - unless only the default argument values are always used during method's invocation.

Is it the case here? ;)

Or do you plan to change it in the next PR?

Reply:
As you supposed correctly - invoking the method will be part of another PR for the `api` repository (together with bumping the gem version). 

It should work even without this as we are only using the default values.